tests/test_cli.py

```diff
@@ -33,12 +34,12 @@ def options(tmp_path):
         "--name",
         "NAME",
         "--path",
-        tmp_path,
+        str(tmp_path.resolve()),
```

Comment:
I thought that making the list all strings made sense as it's easier to interpret what we expect.

Reply:
True, but we might need to see how we test the module's ability to take `pathlib.Paths` or `os.dir objects` - or is that more a functionality test?